scripts/pre-push

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+
+# INSTRUCTIONS:
+# This script will run the pre-push checks in the given order
+# Ktlint
+#
+
+# this will run the ktlint check script
+bash kotlin-lint-check
+
+# TODO() Add Bazel Linter to the project #1736
+# TODO() Add Java Linter to the project #1735
+# TODO() Add an XML Linter to the project #970
+# TODO() Add Buf linter
```

Comment:
TODOs should be of the format ``TODO(#1234): Actionable item.``

Please fix. Each TODO should have an issue associated with it.

Reply:
done